### PR TITLE
chore: 1.9 release notes

### LINKED ISSUE #6561: Release Notes for v1.9
### Is your enhancement related to a problem? Please describe

Add new release notes for v1.9

### Describe the solution you'd like

provide release notes PR

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Release Notes for 1.9 release

### What issues does this PR fix or reference?

Fixes #6561 

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/blog/2024-04-05-release-1.9.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.9 Release! 🎉
+
+![Podman-desktop-1-9-hero](img/podman-desktop-release-1.9/juggling.png)
+
+We've got a new release with a ton of seal appeal! This release introduces:
+
+- **Podman 5 !** [Podman 5.0.1](https://github.com/containers/podman/releases/tag/v5.0.1) as an experimental feature for existing users or set as default for new users.
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: 'as an experimental feature for existing users or set as default for new users.' is a bit hard to parse. Maybe 'for new users (and as an experimental upgrade for 4.9 users)'?

File: website/blog/2024-04-05-release-1.9.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+Moving to Podman v5 implies to optionally save images and then delete previous machines and create a new one.
+
+#### Important Steps before updating
+
+Prioritize data backup by using the save feature in the `Image Lists` section. This feature allows you to back up your images and restore them once you have a new Podman machine.
+
+When prompted to update, confirm to remove all existing data from your machines.
+
+For more details on the save/load feature, refer to the release notes.
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: This is a blog, but I think to many people these are our release notes. Maybe add a link?

File: website/blog/2024-04-05-release-1.9.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+![v5 update](img/podman-desktop-release-1.9/podman-update-to-v5.png)
+
+**_NOTE:_** On macOS you will notice a lot of performance enhancements as Apple Hypervisor is used instead of QEMU for the Podman machine. Expect better performance on the file sharing.
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: I think we should start the v5 section with this line and the next one - i.e. Podman v5 is out, it's awesome, comes with huge improvements for Mac - but is incompatible so we're starting the roll out for new users only as we test and improve migration.

File: website/blog/2024-04-05-release-1.9.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+Here is a blog post introducing Podman v5 https://blog.podman.io/2024/03/podman-5-0-has-been-released/
+
+#### Enabling experimental flag
+
+If you're running a Podman version 4, to see the update to version 5 you need to enable a flag `v5` in Podman extension preferences.
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: It's not clear to the user what this is going to do. Maybe "Are you using Podman v4 but eager to migrate? Enable the Podman 'v5' experimental flag in Settings > Preferences > Podman. This will make the Upgrade option available on the Dashboard."

File: website/blog/2024-04-05-release-1.9.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+
+![Remove previous Podman machines](img/podman-desktop-release-1.9/podman-require-onboarding-2.png)
+
+### Podman 4.9.4
+
+If you want to stay on Podman v4.x, there is a new update for some bugs including a CVE/security issue.
+
+### Save/Load Images or Export/Import Containers
+
+Because Podman v5 is not able to read Podman v4 machines, before updating, backup images that you need to keep. Maybe you don't need to save images if they are all
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: Sentence a little hard to read and backup/save used interchangeably. Suggest: "Podman v5 is not able to read Podman v4 machines, so before updating you'll need to backup images that you need to keep. You don't need to backup images that are available on remote registries or transient."

File: website/blog/2024-04-05-release-1.9.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+
+![Select containers to export](img/podman-desktop-release-1.9/podman-export-1.png)
+
+![Export containers](img/podman-desktop-release-1.9/podman-export-2.png)
+
+Import containers using the Load button from the image list.
+
+![Select files to import](img/podman-desktop-release-1.9/podman-import-1.png)
+
+![Import images](img/podman-desktop-release-1.9/podman-import-2.png)
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md:107]: Import containers

File: website/blog/2024-04-05-release-1.9.md [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+
+**_NOTE:_** Exporting the filesystem of containers only export the content of the filesystem. Importing will result in a container without any commands, so this might not be what you expect.
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: export > exports

File: website/blog/2024-04-05-release-1.9.md [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+Please prioritize the usage of image saving/loading over container export/import
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: Missing .

File: website/blog/2024-04-05-release-1.9.md [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+
+We continued spent a lot of time adding new extension API to give upcoming extensions
+more capabilites and even better integration into 🦭 Podman Desktop:
+
+- feat: add navigateToAuthentication method to navigation API [6603](https://github.com/containers/podman-desktop/pull/6603)
+- feat: add secrets handling to extensionContext in extension api [6423](https://github.com/containers/podman-desktop/pull/6423)
+- feat: add sign in button for auth providers w/ the only auth session request [6446](https://github.com/containers/podman-desktop/pull/6446)
+- feat: support for webview reveal [6546](https://github.com/containers/podman-desktop/pull/6546)
+
+Also we published a test framework to test extension in separate repositories
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: extension > extensions

File: website/blog/2024-04-05-release-1.9.md [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+image: /img/blog/podman-desktop-release-1.9/juggling.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.9 Release! 🎉
+
+![Podman-desktop-1-9-hero](img/podman-desktop-release-1.9/juggling.png)
+
+We've got a new release with a ton of seal appeal! This release introduces:
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: I don't have any better suggestions, but IMHO we've used the 'seal appeal' a little too often.

File: website/blog/2024-04-05-release-1.9.md [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+![Select files to import](img/podman-desktop-release-1.9/podman-import-1.png)
+
+![Import images](img/podman-desktop-release-1.9/podman-import-2.png)
+
+**_NOTE:_** Exporting the filesystem of containers only export the content of the filesystem. Importing will result in a container without any commands, so this might not be what you expect.
+Please prioritize the usage of image saving/loading over container export/import
+
+### Terminal in Kubernetes Pods
+
+In the previous release, we introduced several new features in Kubernetes, but one notable addition was missing. With the 1.9 release, we're excited to announce that you can now connect to the terminal of a pod.
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md]: Remove first comma, it breaks up the sentence too much.

File: website/blog/2024-04-05-release-1.9.md (hunk 6 above)
Comment (benoitf) [reply website/blog/2024-04-05-release-1.9.md:107]: it's really an image, the container is exported as an image

File: website/blog/2024-04-05-release-1.9.md (hunk 10 above)
Comment (benoitf) [reply website/blog/2024-04-05-release-1.9.md]: changed

File: website/blog/2024-04-05-release-1.9.md [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+
+On macOS there are performance enhancements as Apple Hypervisor is used instead of QEMU for the Podman machine. Also expect better performance on the file sharing.
+
+That said, one issue is that Podman machine version 4 is not compliant at all with a Podman machine version 5.
+
+This is why we're rolling out Podman version 5 by default only to new users to improve the migration from Podman Desktop side.
+
+Moving to Podman v5 implies to optionally save images and then delete previous machines and create a new one.
+
+For Podman version 4 users, Podman version 5 is accessible using an experimental flag.
```
Comment (jeffmaury) [inline website/blog/2024-04-05-release-1.9.md]:
```suggestion
For Podman version 4 users, migration to Podman version 5 is accessible using an experimental flag.
```

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by lstocchi — APPROVED** [review]
LGTM 🚀

**Comment by benoitf** [comment]
@deboer-tim updated

File: website/blog/2024-04-05-release-1.9.md (hunk 12 above)
Comment (benoitf) [inline website/blog/2024-04-05-release-1.9.md]:
```suggestion
For Podman version 4 users, migrating to Podman version 5 is accessible using an experimental flag.
```

File: website/blog/2024-04-05-release-1.9.md [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+image: /img/blog/podman-desktop-release-1.9/juggling.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.9 Release! 🎉
+
+![Podman-desktop-1-9-hero](img/podman-desktop-release-1.9/juggling.png)
+
+This release introduces: 🦭 a splash of innovation, a wave of excitement, and an ocean of possibilities!"
```
Comment (slemeur) [inline website/blog/2024-04-05-release-1.9.md]: love it!

File: website/blog/2024-04-05-release-1.9.md [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+
+## Release Details
+
+### Podman v5 (5.0.1)
+
+Podman version 5 is out! [see blog post introducing Podman v5](https://blog.podman.io/2024/03/podman-5-0-has-been-released/)
+
+On macOS there are performance enhancements as Apple Hypervisor is used instead of QEMU for the Podman machine. Also expect better performance on the file sharing.
+
+That said, one issue is that Podman machine version 4 is not compliant at all with a Podman machine version 5.
```
Comment (slemeur) [inline website/blog/2024-04-05-release-1.9.md]:
```suggestion
That said, Podman machine version 5 is introducing changes that are not compliant with podman machine version 4. 
```

File: website/blog/2024-04-05-release-1.9.md [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+
+This is why we're rolling out Podman version 5 by default only to new users to improve the migration from Podman Desktop side.
+
+Moving to Podman v5 implies to optionally save images and then delete previous machines and create a new one.
```
Comment (slemeur) [inline website/blog/2024-04-05-release-1.9.md]: I would invert with the sentence that is following after

File: website/blog/2024-04-05-release-1.9.md [hunk 16]
```diff
@@ -0,0 +1,224 @@
...
+
+For Podman version 4 users, migrating to Podman version 5 is accessible using an experimental flag.
+
+#### Important Steps before updating
```
Comment (slemeur) [inline website/blog/2024-04-05-release-1.9.md]:
```suggestion
#### Important steps before updating
```

File: website/blog/2024-04-05-release-1.9.md [hunk 17]
```diff
@@ -0,0 +1,224 @@
...
+
+On macOS there are performance enhancements as Apple Hypervisor is used instead of QEMU for the Podman machine. Also expect better performance on the file sharing.
+
+That said, one issue is that Podman machine version 4 is not compliant at all with a Podman machine version 5.
+
+This is why we're rolling out Podman version 5 by default only to new users to improve the migration from Podman Desktop side.
+
+Moving to Podman v5 implies to optionally save images and then delete previous machines and create a new one.
+
+For Podman version 4 users, migrating to Podman version 5 is accessible using an experimental flag.
```
Comment (slemeur) [inline website/blog/2024-04-05-release-1.9.md]: link to the following section?

**Review by slemeur — COMMENTED** [review]
Good job on the release note!

File: website/blog/2024-04-05-release-1.9.md (hunk 13 above)
Comment (deboer-tim) [reply website/blog/2024-04-05-release-1.9.md]: +1

Comment (deboer-tim) [reply website/blog/2024-04-05-release-1.9.md]: unmatched " at the end though

File: website/blog/2024-04-05-release-1.9.md (hunk 6 above)
Comment (deboer-tim) [reply website/blog/2024-04-05-release-1.9.md:107]: I commented b/c the png is titled Import Containers but the markdown title says Images

File: website/blog/2024-04-05-release-1.9.md [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.9/juggling.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.9 Release! 🎉
+
+![Podman-desktop-1-9-hero](img/podman-desktop-release-1.9/juggling.png)
```
Comment (deboer-tim) [inline website/blog/2024-04-05-release-1.9.md:15]: Is this us juggling release priorities? ;-)

**Review by deboer-tim — APPROVED** [review]
A couple minor comments from Stevan, then good to go.

**Comment by benoitf** [comment]
@slemeur updated
